docs/pages/includes/database-access/auto-user-provisioning/troubleshooting-remote-username.mdx

```diff
@@ -0,0 +1,12 @@
+You may encounter the following error when connecting to a database in a remote
+cluster:
+```shell
+> tsh db connect mysql --db-name <database> example
+ERROR: please use your mapped remote username ("remote-<your-teleport-username>-<root-cluster-name>") to connect instead of "<database-user>"
+```
```

Comment:
I wonder if we can simply... ignore the username the user has sent and force the client to use the expected one?

Reply:
we can, and I've suggested a similar thing before (if they specify their own username, just wrap it with `remote-<name>-<cluster>`), but then our CLI is lying to the user about what db user they are actually connecting as

Reply:
Perhaps we can tweak `tsh` to recognise this case and issue a warning to the user?

Reply:
`tsh db connect` does detect this case and makes the correct database username, when `--db-user` is not specified. I think this error will most likely appear when using GUI clients through `tsh proxy db` and the GUI will get  see this error from the backend.